demo/src/pick_place_task.cpp

```diff
@@ -490,7 +490,7 @@ bool PickPlaceTask::plan() {
 	ROS_INFO_NAMED(LOGNAME, "Start searching for task solutions");
 	int max_solutions = pnh_.param<int>("max_solutions", 10);
 
-	return task_->plan(max_solutions);
+	return task_->plan(max_solutions) == moveit_msgs::MoveItErrorCodes::SUCCESS;
```

Comment:
Can't you stick with the old code here, because the result is cast to bool automatically?

Reply:
For some reason I'm getting `No viable conversion from returned value of type 'moveit::core::MoveItErrorCode' to function return type 'bool' explicit conversion function is not a candidate` with the old version

Reply:
Try with explicit conversion to bool then?
```suggestion
	return static_cast<bool>(task_->plan(max_solutions));
```